lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Context/ContextManager.cs

```diff
@@ -382,5 +405,14 @@ private void HandleContextException(ContextClientCodeException e)
             LOGGER.Log(Level.Error, "Sending Heartbeat for a failed context: {0}", contextStatusProto);
             _heartBeatManager.OnNext(contextStatusProto);
         }
+
+        private static byte[] SerializeException(Exception ex)
```

Comment:
This strikes me as something we might want to move to REEF Utilities, as we probably need it in lots of places?